### PR TITLE
Get specific place

### PR DESCRIPTION
**Pull request by RQ16**
#13 
Get specific place information

### CODE REVIEW

**Comment by m4v15** [comment]
You've assigned me to review but it's still a WIP, right?

We need to do the language and text direction for this stuff as wel

File: views/place.hbs [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<ul class="specific-place">
+  <li>Name:
```
Comment (m4v15) [inline views/place.hbs]: don't use Name - use `localLang.name` or like in views/places

**Comment by RQ16** [comment]
@m4v15 I have added the text direction and language stuff- It is ready for reviewing.

File: views/place.hbs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  <li>{{localLang.placeHours}}:
+    {{place.local.openingHours}}
+  </li>
+
+  <li>{{localLang.placeLocation}}:
+    {{place.location}}
+  </li>
+
+  <li>{{localLang.placeImage}}:
+    <a href="{{place.imageUrl}}">View</a>
```
Comment (m4v15) [inline views/place.hbs]: Why are these links and not images?

File: views/place.hbs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  </li>
+
+  <li>{{localLang.placeWebsite}}:
+    <a href="{{place.website}}">View</a>
```
Comment (m4v15) [inline views/place.hbs]: Yeah I'd maybe change these - the image could be an image and this could have more informative link text

**Review by m4v15 — CHANGES_REQUESTED** [review]

**Comment by m4v15** [comment]
I don't think we have to rush to get this through before this sprint planning tbh

File: views/place.hbs [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  <li>{{localLang.placeAddress}}:
+    {{place.local.address}}
+  </li>
+
+  <li>{{localLang.placeHours}}:
+    {{place.local.openingHours}}
+  </li>
+
+  <li>{{localLang.placeLocation}}:
+    {{place.location}}
```
Comment (m4v15) [inline views/place.hbs]: This could be a map? Having co-ordinates isn't massively helpful

**Comment by Suuuuha** [comment]
@m4v15 changes have been made!

**Review by m4v15 — APPROVED** [review]

**Comment by m4v15** [comment]
looking good 🔥

**Pull request merged**